### PR TITLE
Stat tracker from csv

### PR DESCRIPTION
**Pull request by Dav1s-Ops**
- Attributes for StatTracker
- Instance test for StatTracker
- attr_readers for class files

### CODE REVIEW

**Review by kcockett — APPROVED** [review]
As we all collaborated together on this, I approve this change

**Review by Gabe-Torres — APPROVED** [review]
Testing on the stat tracker spec is looking solid. Good call on the attr_reader too!

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -1,41 +1,43 @@
...
     stat_tracker = StatTracker.new
     
-    game_hash[:games] = stat_tracker.game_generator(locations)
-    game_hash[:teams] = stat_tracker.teams_generator(locations)
-    game_hash[:game_teams] = stat_tracker.game_teams_generator(locations)
-    game_hash
+    stat_tracker.game_generator(locations)
+    stat_tracker.teams_generator(locations)
+    stat_tracker.game_teams_generator(locations)
+    return stat_tracker
```
Comment (WZale) [inline lib/stat_tracker.rb:20]: THAT'S the way to make the instance variables accessible? Interesting. Glad you all figured this out!

File: spec/stat_tracker_spec.rb [hunk 2]
```diff
@@ -17,6 +17,15 @@
   end
```
Comment (WZale) [inline spec/stat_tracker_spec.rb:18]: Oh good, now we can keep things strictly TDD from here on out.

File: lib/team.rb [hunk 3]
```diff
@@ -1,9 +1,9 @@
 class Team
-  # attr_reader :team_id, 
-  #             :franchise_id, 
-  #             :team_name, 
-  #             :abbreviation, 
-  #             :stadium
+  attr_reader :team_id,
```
Comment (WZale) [inline lib/team.rb:2]: I did wonder if we were going to need these.

**Pull request merged**